cmd/did-method-rest/go.mod

```diff
@@ -8,7 +8,6 @@ replace github.com/trustbloc/trustbloc-did-method => ../..
 
 require (
 	github.com/gorilla/mux v1.7.4
-	github.com/hyperledger/aries-framework-go v0.1.3-0.20200430150510-b5d30427f098
```

Comment:
Why did go mod tidy remove this? @fqutishat

Reply:
not used in folder cmd